packages/cardpay-sdk/sdk/utils/conversions.ts

```diff
@@ -8,6 +8,12 @@ import { networkName } from './general-utils';
 import BN from 'bn.js';
 import { BaseProvider } from '@ethersproject/providers';
 
+export interface GasPrice {
+  slow: BN,
+  standard: BN,
+  fast: BN
+}
+
```

Comment:
supernit: if all the values have  the same type you could do something like:

```ts
type GasPrice = Record<'slow' | 'standard' | 'fast', BN>;
```

Reply:
updated